amy/autoemails/models.py

```diff
@@ -386,6 +391,24 @@ class RQJob(CreatedUpdatedMixin, models.Model):
         default="",
         verbose_name="Mail recipients",
     )
+    interval = models.IntegerField(
+        blank=True,
+        null=True,
+        verbose_name="Interval in seconds",
+    )
+    result_ttl = models.IntegerField(
+        blank=True,
+        null=True,
+        verbose_name="Result TTL",
+    )
```

Comment:
@pbanaszkiewicz  It's not currently overriding any defaults. The repeated jobs have a TTL of -1 by default. So I'm not changing anything there. I can't seem to recall why I surfaced this variable. I don't make any use of it anywhere. I'll just remove it.

Reply:
Eh on second thought it's not bad information to have if you're planning on changing things it might be fine to just keep. But no defaults are changed here.